tfidf.py

```diff
@@ -0,0 +1,80 @@
+import os
+import argparse
+from sklearn.feature_extraction.text import CountVectorizer
+from sklearn.feature_extraction.text import TfidfTransformer
+import pickle as pkl
+from clams_utils.aapb.newshour_transcript_cleanup import file_cleaner
+
+def lists_generator(directory):
```

Comment:
Since inside this function, we have a call to `file_cleaner` which only works with NH files we have, I think it's safer to rename the function (also rename the argument name) to make it more clear that it expects the input directory to be a collection of NH transcript files.